src/api-reference/user/index.markdown

```diff
@@ -27,8 +25,7 @@ This resource allows you to get profile information for a given user. If a reque
 
 Name | Type | Format | Description
 -----|------| ------ | --------------
-`loginID`|`string` | | The URL-encoded Concur login of the user. Optional.
-
+`loginID`|`string` | | The URL-encoded SAP Concur login of the user. Optional.
```

Comment:
We need to clean up whitespace and make sure to denote empty cells intentionally with `|-|`.

Reply:
Excellent, thanks.